lib/smart_proxy_pulp_plugin/disk_usage.rb

```diff
@@ -29,11 +31,12 @@ def find_df
     end
 
     def command
-      [command_path, "-B", "#{size}", *path]
+      ENV['LC_ALL'] = 'C'
```

Comment:
if its a copy paste, it should include a copyright or credit.

On Sun, May 29, 2016 at 3:39 PM, Shlomi Zadok <notifications@github.com>
wrote:

> In lib/smart_proxy_pulp_plugin/disk_usage.rb
> <https://github.com/theforeman/smart-proxy-pulp/pull/5#discussion_r65004055>
> :
>
> > @@ -29,11 +31,12 @@ def find_df
> >      end
> >
> >      def command
> > -      [command_path, "-B", "#{size}", *path]
> > +      ENV['LC_ALL'] = 'C'
>
> Added as a separate commit, lmk what u think
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/theforeman/smart-proxy-pulp/pull/5/files/6002247d175d928b08c5cdbb0d4628dd78ad1317#r65004055>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AABOx82Rd1WkQODaukmthaRknJLjvSb5ks5qGYkIgaJpZM4IX59F>
> .
>